### ISSUE TITLE
Form is not processed, redirect to index.php on SSL-Page

### ISSUE THREAD

**Issue by instantflorian** [WebsiteBaker-modules/mpform#6]
WBCE CMS 1.3, mpform 1.3.8 / 1.3.11 (tested both)

Whether entering some data or not, the form is not processed when clicking on "submit", but the user is redirected to the domain.tld/index.php of the site. The entered data is lost (is not stored in the database, appears not in the backend). If there are mandantory fields empty, no message appears, but also a redirect to the /index.php.

It seems as if this might only be happening on SSL encrypted connections and/or only on sites which are hosted at a certain company. On another test page (another hoster, no SSL) the error does not appear.

I can give you backend / FTP access to the site where the error appears if you want to. 

Edit: must be sth. about WBCE CMS 1.3, does not happen with 1.2 :(

**Comment by mrbaseman** [comment]
I had a look at the code of mpform and the changes between WBCE 1.2 and 1.3 and did not (yet) find a clear explanation. The behavior (redirect to the pages-directory) is typical for cases in which mpform assumes that the submitted data was spoofed (i.e. the matching entries in the session are not set) or the submitted data was filled in by a robot (which did enter something into the ASP fields normally not visible to a human. I assume that the problem is related to the changes of the session handling that has been changed in WBCE between 1.2 and 1.3. 
The strange thing is that it is related to https vs. http. I did not find anything protocol related in this area. But on a second reading it might also be that it depends on the hoster. What else is different between the two hosters?
mpform stores a submission_id for each section in the $_SESSION array and expects to find this submission id there again when processing the posted form data. Is this still supposed to work in WBCE 1.3? How could this work with http but fail with https (or: might there be a hosting platform dependency in the session handling, which did not affect users of WBCE 1.2 but strikes under 1.3)?
